vagrant_variables.yml.atomic

```diff
@@ -23,6 +23,9 @@ disks: "[ '/dev/sdb', '/dev/sdc' ]"
 
 eth: 'enp0s8'
 vagrant_box: centos/atomic-host
+#The sync directory changes based on vagrant box
```

Comment:
Do you mind putting a space between `#` and the first letter of the word? It's easier to read.